packages/api/tests/api/contributions.test.ts

```diff
@@ -0,0 +1,163 @@
+import { Collection } from '@mikro-orm/core';
+import { contributions } from '../../src/api/contributions';
+import { Contribution } from '../../src/entities/Contribution';
+import { User } from '../../src/entities/User';
+import logger from '../../src/logger';
+import { testHandler } from '../testUtils/testHandler';
+
+const sampleUser: Partial<User> = {
+  name: 'Bill Nye',
+  pronouns: 'he/him',
+  schoolName: 'Science School',
+  assign: jest.fn(),
+  isAdmin: false,
+  githubId: '234234',
+  id: '1',
+  contributionList: new Collection<Contribution>(this),
```

Comment:
Unless I'm missing something, we shouldn't be using this `Collection` thing anywhere... I could be wrong, but it looks like we're overly complicating this by not just using an array (see my comment on the entity)